pkg/ccl/backupccl/testdata/backup-restore/multiregion

```diff
@@ -65,6 +65,27 @@ RESTORE FROM LATEST IN 'nodelocal://1/full_cluster_backup/';
 pq: detected a mismatch in regions between the restore cluster and the backup cluster, missing regions detected: us-east-1, us-west-1.
 HINT: there are two ways you can resolve this issue: 1) update the cluster to which you're restoring to ensure that the regions present on the nodes' --locality flags match those present in the backup image, or 2) restore with the "skip_localities_check" option
 
+exec-sql
```

Comment:
nit: I'd run the cluster restore first, then the database restore with  the `new_db_name` option. Else, the cluster restore may flake because it detects database that's in the process of getting dropped, even after the DROP DATABASE command returns.

Reply:
Good point let me change this